### DISCUSSION TITLE
MaxListenersExceededWarning

### DISCUSSION THREAD

**Discussion by coreTech51** [kuestess/homebridge-platform-insteonlocal#313] [Q&A]
Anyone know how to fix this? I keep getting this resulting in an error.

Thanks,
Aaron

(node:31331) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 107 connect listeners added to [Insteon]. Use emitter.setMaxListeners() to increase limit

(Use `node --trace-warnings ...` to show where the warning was created)

[31/01/2024, 10:54:09] Homebridge v1.7.0 (HAP v0.11.1) (InsteonLocal) is running on port 56513.

[1/31/2024, 10:54:19 AM] [InsteonUI] Error getting Hub info

**Comment by kuestess** [comment]
@coreTech51 See here: [https://github.com/kuestess/homebridge-platform-insteonlocal/issues/199](https://github.com/kuestess/homebridge-platform-insteonlocal/issues/199)
